test/LibraryManager.IntegrationTest/AddClientSideLibrariesFromUITests.cs

```diff
@@ -12,20 +9,17 @@ namespace Microsoft.Web.LibraryManager.IntegrationTest
     [TestClass]
     public class AddClientSideLibrariesFromUITests : VisualStudioLibmanHostTest
     {
-        private string _initialLibmanFileContent;
         private string _pathToLibmanFile;
```

Comment:
It would be swell if these private fields were inherited from the parent class.  They don't seem specific to this class at all.

Reply:
updated in iteration 2 to inherit from parent class

---
In reply to: [208032751](https://github.com/aspnet/LibraryManager/pull/341#discussion_r208032751) [](ancestors = 208032751)